### PR TITLE
Mvh stattracker

### PR DESCRIPTION
**Pull request by mikez321**
Fixed all lower case CSVs
Expanded StatTracker to open and populate all three data sets
All tests pass
Rake file works
All local and instance variables named to convention

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -1,19 +1,24 @@
-require 'CSV'
```
Comment (aperezsantos) [inline lib/stat_tracker.rb:1]: Just to clarify, stat_tracker does not need csv required?

File: lib/game_teams_collection.rb [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
-  def initialize(game_teams_csv)
-    @game_teams = create_game_teams(game_teams_csv)
+  def initialize(game_teams_file_path)
+    @game_teams = create_game_teams(game_teams_file_path)
   end
 
-  def create_game_teams(game_teams_csv)
-    game_teams_data = CSV.read(game_teams_csv, headers: true, header_converters: :symbol)
-    game_teams = game_teams_data.map do |row|
+  def create_game_teams(game_teams_file_path)
```
Comment (colintalex) [inline lib/game_teams_collection.rb:9]: This makes the readability much easier

File: test/game_test.rb [hunk 3]
```diff
@@ -1,9 +1,7 @@
-require 'SimpleCOV'
```
Comment (aperezsantos) [inline test/game_test.rb:1]: Since we need this at some point, maybe we need to figure out how to run rake without SimpleCov interfering?

**Review by aperezsantos — APPROVED** [review]
Great work! I just added a few questions for clarification.

**Review by colintalex — APPROVED** [review]
Looks good to me, the changes we intended came out properly.

File: test/game_test.rb (hunk 3 above)
Comment (colintalex) [reply test/game_test.rb:1]: Agreed, I think we can make a card for this on our project board so it doesn't get overlooked

File: lib/stat_tracker.rb (hunk 1 above)
Comment (mikez321) [reply lib/stat_tracker.rb:1]: That was removed because we moved the CSV converting to the collections rather than the stattracker.

**Pull request merged**